### DISCUSSION TITLE
Redirect for www.delph-in.net?

### DISCUSSION THREAD

**Discussion by emilymbender** [delph-in/docs#64] [Team Posts]
Two question in the context of a couple of journal papers getting polished, which point (like many before) to www.delph-in.net, which currently redirects to http://moin.delph-in.net/wiki/FrontPage

1) Will that domain persist?
2) Should it now point to https://github.com/delph-in/docs/wiki

Thanks :)

**Comment by goodmami** [comment]
I think I also proposed creating a non-wiki GitHub-hosted page at https://delph-in.github.io which would be what delph-in.net points to. We could then have a static site without all the GitHub chrome present on the wiki. For example, Bond Lab's website is hosted this way: https://bond-lab.github.io/

**Comment by oepen** [comment]
Hiya,

On 11 May 2022, at 00:47, emilymbender ***@***.***> wrote:

  *   Will that domain persist?
  *   Should it now point to https://github.com/delph-in/docs/wiki

Yes to both from my point of view.  I expect UiO to host a web server at that domain for many years to come, and I should really sit down and update the various layers of rewrite rules to point to the new GitHub-hosted wiki.

Best, oe
